src/main/java/jp/co/yahoo/yosegi/message/objects/BooleanObj.java

```diff
@@ -109,16 +104,32 @@ public void setBytes( final byte[] data ) throws IOException {
 
   @Override
   public void setBytes(
-      final byte[] data ,
-      final int start ,
-      final int length ) throws IOException {
-    value = 
+      final byte[] data,
+      final int start,
+      final int length) throws IOException {
+
+    /*
+     * Original code is not use start, but it seems start as buffer start point,
```

Comment:
It is a bug. Please delete the original code as it is as the comment.

Reply:
remove the comment